src/wallet/rpcwallet.cpp

```diff
@@ -3401,6 +3401,8 @@ UniValue z_listreceivedaddress(const UniValue& params, bool fHelp,const CPubKey&
         "       \"saplingReceived\": [{               An Array of utxos/notes received for sapling addresses\n"
         "           \"address\": \"zeroaddress\",         (string) Shielded address (z-address)\n"
         "           \"amount\": x.xxxx,                 (numeric) Value of output being received " + CURRENCY_UNIT + ", positive for receives\n"
+        "           \"memo\": xxxxx,            (string) hexademical string representation of memo field\n"
+        "           \"memoStr\" : \"memo\",             (string) Only returned if memo contains valid UTF-8 text.\n"
```

Comment:
We should always return the key, and it should be empty if there is no valid string representation